.changeset/nice-panthers-doubt.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/stylelint-polaris': major
```

Comment:
I don't mind it being `minor` because new errors are not being added.

Reply:
Since this discussion, we decided that if rule changes have the potential to error in CI then we will do a major version upgrade for `stylelint-polaris` and this PR definitely will.